src/xrdsvc/SsiService.cc

```diff
@@ -168,6 +168,9 @@ SsiService::SsiService(XrdSsiLogger* log) {
                 memman::MemMan::create(memManSize, workerConfig->getMemManLocation()));
     } else if (cfgMemMan == "MemManNone") {
         memMan = make_shared<memman::MemManNone>(1, false);
+    } else if (cfgMemMan == "MemManNoneRelaxed") {
+        bool const alwaysLock = true;
+        memMan = make_shared<memman::MemManNone>(1, alwaysLock);
```

Comment:
The naming here seems counter intuitive. I'm guessing alwaysLock = true, means consider it locked requardless of if it could be locked? I think some comments explaining what is going on with the different modes will be helpful.

Reply:
Completely agree with you. However, the variable was named to match the name of the corresponding formal parameter of the constructor (see class `memman::MemManNone`). This parameter and the class existed before. So, I decided not to change anything.